<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>401</width>
    <height>321</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>40</y>
     <width>301</width>
     <height>241</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>Title</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>url</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Date</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>10</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Avenir</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Reddit Scraper</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
